### DISCUSSION TITLE
Coding Club - Rolling Notes

### DISCUSSION THREAD

**Discussion by ran-codes** [Drexel-UHC/data-science#51] [Meeting Agenda/Notes]
In person meetings tracking development of 'coding club' and https://github.com/Drexel-UHC/analytics-corner/issues/19

**Comment by ran-codes** [comment]
# Nov 29, 2023

## Agenda

- [ ] Review overall goals
   - [ ] Data science capacity building at Drexel: Reproducible research/PHDSAG request (https://github.com/Drexel-UHC/analytics-corner/issues/19)
   - [ ] Support P20/c-SALURBAL works
- [ ] Brainstorm official points of contact
   - [ ] c-SALURBAL Aim 3 leads: Mariana Lazo + Maria Kroker
   - [ ] DMC leads/coordinator: Usama Bilal, Kari Moore
   - [ ] Dornsife: Aritra Holder, Loni Tab
   - [ ] Any others? How/who should we approach these points of contact.
- [ ] Brainstorm of skills to cover 
   - [ ] overall goal is to build critical skillskets to enable 'coding club'  where the focus on live-reproduction of analyssis
   - [ ] https://github.com/Drexel-UHC/data-science/discussions/17
   - [ ]  https://github.com/Drexel-UHC/data-science/discussions/32
   - [ ] come up with a list of skills this group is interested in.
       - [ ] GitHub 
          - [x] Desktop software
          - [x] set up repository
          - [x] clone a repository
          - [x] Pulling and Pushing
          - [ ] Project Management (important for collaborative)
          - [ ] Branches (optional ... but improtant for collaboration)
        - [ ] Computation
          - [ ] Rstudio best practice and workflows  (Brian's class, Rogen peng, michigan)
             - [ ] maybe veryeon share 5 miuntes on how they use strstio and organiz projects
- [ ] Discuss approach
    - [x] kitchen sink approach
    - [ ] skill based approach 
    - [ ] compile list of repositories to present so far see [request](https://github.com/Drexel-UHC/analytics-corner/issues/19)
- [ ] discuss next steps

## Action Items

**Reply by ran-codes** [reply]
## Action Items

- [ ] each person compile a list of manuscript repostiroies for use to work through and reproduce 
   - [ ]  comment here with the repo
- [ ] prepare a 5 miunte walkthrough about how you use Rstudio do/organize a manuscript/analysis/data-prep
- [ ] @ran-codes liase with Hal about PHDSAG
- [ ] @juruchi  @ran-codes to prep a project mangament training for this group.

**Comment by juruchi** [comment]
Manuscript repositories:
- https://github.com/confunguido/public_health_assessment_dengue_vaccine
- https://github.com/zabore/manuscript-code-repository/tree/master
- https://github.com/PombertLab/Publication_scripts/tree/master
- https://github.com/shilaan/example-manuscripts
- https://github.com/ddsjoberg/Manuscript-Code-Share

Rstudio use walkthrough: 
[Uruchima_How I Use R.pptx](https://github.com/Drexel-UHC/data-science/files/13606027/Uruchima_How.I.Use.R.pptx)
